packages/service-clients/odsp-client/src/odspAudience.ts

```diff
@@ -10,14 +10,14 @@ import { type OdspMember, OdspUser } from "./interfaces";
 export function createOdspAudienceMember(audienceMember: IClient): OdspMember {
 	const user = audienceMember.user as OdspUser;
 	assert(
-		user.name !== undefined || user.email !== undefined,
+		user.name !== undefined || user.id !== undefined || user.oid !== undefined,
```

Comment:
```suggestion
		user.name !== undefined || user.email  !== undefined || user.oid !== undefined,
```

Reply:
`user.id` is the same as `user.email` if I understand Sonali's PR description correctly, and to resolve the redundancy you pointed out we can only really get rid of `user.email` (at least if we stick with `IUser` which is where `user.id` is coming from).

Reply:
just don't use IUser.id, it makes the code confusing as its is not what would be expected, and nothing forces its usage since it is redundant via the email property anyway